src/sentry/api/helpers/group_index.py

```diff
@@ -493,6 +494,51 @@ def wrapper(self, request, *args, **kwargs):
     return inner
 
 
+def get_current_release_version_of_group(project_id, group_id):
+    """
+    Function that returns the latest semver release version linked to a Group or the
+    `current_release_version` for GroupResolution
+    Inputs:
+        * project_id
+        * group_id
+    Returns:
+        current_release_version
+    """
+    try:
+        # This represents the latest semver version where this group occurs in
+        current_release_version = list(
+            Release.objects.raw(
+                """
+                    SELECT sr.*,
+                          CASE
+                            WHEN sr.prerelease = '' THEN 1
+                            ELSE 0
+                          END AS prerelease_case
+                    FROM sentry_release as sr
+                    INNER JOIN (
+                        SELECT release_id
+                        FROM sentry_grouprelease
+                        WHERE project_id = %s
+                        AND sentry_grouprelease.group_id = %s
+                        ORDER BY release_id DESC
+                    ) AS sgr ON (sr.id = sgr.release_id)
```

Comment:
What's the benefit of joining to a subquery here? I don't think the order by is helping us at all, unless we're going to limit to a subset of rows